spec/Overview-WebCryptoAPI.xml

```diff
@@ -10083,8 +10083,8 @@ required <a href="#dfn-CryptoKey">CryptoKey</a> <dfn id="dfn-EcdhKeyDeriveParams
                       <ol>
                         <li>
                           <p>
-                            Perform the ECDH primitive specified in <a href="#X9.63">X9.63</a> Section
-                            5.4.1 with <var>key</var> as the EC private key <var>d</var> and the EC public
+                            Perform the ECDH primitive specified in <a href="#RFC6090">RFC6090</a> Section
+                            3 with <var>key</var> as the EC private key <var>d</var> and the EC public
```

Comment:
I briefly debated between using section 3 and section 4 at this point. I was thinking in terms of the math as being the primitive rather than the other information.  I would have no problems switching this.

You are right - I did not read quite deeply enough on this pass that we need to say that we are using the compact notation here.  I was assuming that had to do with point representation so I kind of skipped it.